static/app/components/events/interfaces/request/richHttpContentClippedBoxBodySection.tsx

```diff
@@ -58,12 +61,21 @@ export function RichHttpContentClippedBoxBodySection({
         );
       }
 
-      default:
+      default: {
+        // FIXME: Is the test case in static/app/components/events/interfaces/request/index.spec.tsx accurate?
+        // `EntryRequest['data']['data']` suggests that data may be [key: string, value: any][], **not** Record<string, any>[].
+        // If this is a real scenario, what is the rule for looking up the relevant data in meta? As a hack, I assume that if
+        // we encounter data as an array, that the same meta structure can be used by all items... which allows the test case
+        // to pass.
+        const wrappedMeta = Array.isArray(data)
+          ? Object.fromEntries(data.map((_: any, i: number) => [i, meta]))
+          : meta;
```

Comment:
For reference the weird case I'm referring to: https://github.com/getsentry/sentry/blob/f418e2dff/static/app/components/events/interfaces/request/index.spec.tsx#L28-L97